posthog/api/action.py

```diff
@@ -210,8 +211,8 @@ def _calculate_people(events: QuerySet):
                     return Response([])
                 filtered_events = process_entity_for_events(entity, team_id=team.pk, order_by=None).filter(filter_events(team.pk, filter, entity))
 
-        people = _calculate_people(events=filtered_events)
-        return Response([people])
+        people = _calculate_people(events=filtered_events, offset=offset)
+        return Response({'result': [people], 'offset': offset + 100})
```

Comment:
It'd be good to make this API consistent with the other APIs in terms of pagination: https://posthog.com/docs/api/api#pagination

Reply:
done!